doc/cephadm/mon.rst

```diff
@@ -38,12 +38,18 @@ hosts to the cluster. No further steps are necessary.
   Cephadm deploys new monitor daemons only on hosts that have IPs
   configured in the configured subnet.
 
+Changing the number of monitors from the default
+================================================
```

Comment:
what do you think of making this sub headlines?

Reply:
I did this originally, but they didn't show up in the sidebar.

I'll toy with this a bit and see if I can make it work.